package/package.json

```diff
@@ -14,12 +14,17 @@
     "url": "https://github.com/nex3"
   },
   "engines": {
-    "node": ">=0.11.8"
+    "node": ">=10.0.0"
   },
   "dependencies": {
     "chokidar": ">=2.0.0 <4.0.0"
   },
   "main": "sass.dart.js",
   "bin": "sass.js",
-  "keywords": ["style", "scss", "sass", "preprocessor", "css"]
+  "keywords": ["style", "scss", "sass", "preprocessor", "css"],
+  "browser": {
```

Comment:
Can you explain what this field is doing? It doesn't seem to match npm's [documentation for this field](https://docs.npmjs.com/files/package.json#browser).

Reply:
https://github.com/defunctzombie/package-browser-field-spec
basically, it's a field used by bundlers to remap module requests inside a package (when bundling it for the web).

many projects use it, such as axios, react, etc.

Reply:
Can you walk me through the use-case here? Dart Sass doesn't support running in the browser, so what benefit does this provide, exactly?

Reply:
I wanted to run render/renderSync inside a Web Worker, in order to compile `.sass/scss` code into `.css`.

With these changes I can bundle it and run it in the browser, so I'm not sure what you mean by "doesn't support".

Reply:
Dart Sass is built with the assumption that built-in Node.js modules will always be available. If they're not, it's liable to crash in unexpected ways if you invoke it wrong. 

If the goal here is to invoke Dart Sass from within the browser, I want to do it right: by addressing #25 and not adding the default filesystem importer in a web context, and documenting that as an option.

Reply:
I understand wanting to provide a full built-in support for running `sass` with different hosts (other than node, i.e. `deno` or other runtimes).

> Dart Sass is built with the assumption that built-in Node.js modules will always be available. If they're not, it's liable to crash in unexpected ways if you invoke it wrong.

Several node APIs (`url`, `Buffer`, `process` etc.) are polyfill-able and are being automatically polyfilled by bundlers out there. The "browser" field enables me to to ignore other ones (`fs`). As long as sass' process doesn't try to use that `fs`, nothing breaks.

My guess was that `chokidar` and `readline` are used for by the CLI, and so are not relevant to a browser-based flow.

In addition, I found that the existing `data` and `importer` options allow me to "connect" my custom (browser-compatible) fs and resolver to sass' process. This enables me to bypass the need for sass to require an fs provider and achieve rendering of `.scss/sass` files in the browser.

imho, this PR is a good first step towards resolving issue #25.

Reply:
Further investigation into @import handling revealed the gaps in terms of custom hosts. I now see what you mean.